docker/docker-compose.yml

```diff
@@ -21,6 +21,7 @@ x-app: &app
     PSQL_HISTFILE: /app/log/.psql_history
     EDITOR: vi
     CHROMEDRIVER_VERSION: 79.0.3945.36
+    BUNDLE_FORCE_RUBY_PLATFORM: true
```

Comment:
This line blows up for me -- something about how true isn't a valid type for a docker environment variable. Does it not do it for you?

Reply:
It works for me, but I did have to `docker-compose build`.  Let me know if that doesn't fix it for you, I have a less good solution we can try.